frontend/src/config/rbd/layers.json

```diff
@@ -6531,5 +6531,77 @@
       }
     ],
     "legend_text": "Joint Food Security & Nutrition Hotspot Analysis conducted by WFP, UNICEF, and partners based on SMART surveys and Cadre Harmonisé (CH) & Integrated Food Security Phase Classification (IPC) analyses"
+  },
+  "google_flood_status_at_gauges": {
```

Comment:
Should this layer be shared ans then just override the necessary param?

Reply:
I haven't implemented a shared layer before and I'm still trying to understand how it works. I see in the README:
> Since many layers are common across multiple countries, we created shared configuration files that any deployment can access. These layers are generated by WFP globally and made available through the Humanitarian Data Cube.

Is the HDC part meaningful? Based on a quick scan, the current shared layers do appear to all be coming from HDC but I don't see why they _need_ to (technologically speaking that is).

Also, @ericboucher, could you point me to examples of shared layers with overrides? I see examples of shared layers with duplicates in countries but I'm not seeing overrides.